system/ee/ExpressionEngine/Core/Core.php

```diff
@@ -306,8 +311,12 @@ public function setTimeLimit($t)
     /**
      * Setup the application with the default provider
      */
-    protected function loadApplicationCore()
+    public function loadApplicationCore()
     {
+        if ($this->application) {
```

Comment:
Should this be `!is_null($this->application)` instead? Maybe we should not rely on object to evaluate to true. But - we're doing that in lots of other places, and it's probably safe to do.

Reply:
@intoeetive I like that, I've pushed a change for this.